src/streaming/text/TextTracks.js

```diff
@@ -437,40 +492,6 @@ function TextTracks(config) {
         }
     }
 
-    // Check that a new cue immediately follows the previous cue
-    function _areCuesAdjacent(cue, prevCue) {
-        if (!prevCue) {
-            return false;
-        }
-        // Check previous cue endTime with current cue startTime
-        // (should we consider an epsilon margin? for example to get around rounding issues)
-        return prevCue.endTime >= cue.startTime;
-    }
-
-    // Check if cue content is identical. If it is, extend the previous cue.
-    function _extendLastCue(cue, prevCue) {
-        if (!settings.get().streaming.text.extendSegmentedCues) {
```

Comment:
This seems to remove the `extendSegmentedCues` functionality that was added to avoid flickering when playing in low latency mode. See #4112 and #4359. Is this functionality preserved? In any case the setting `extendSegmentedCues` is not used anymore. @bbert @mattjuggins fyi

Reply:
I hadn't understood the point of that code, I meant to ask you before creating the PR, but ended up forgetting. I restored the functionality now.